### PR TITLE
[TRANSLATION] Russian translation browserui

### PR DESCRIPTION
**Pull request by A-Rafael**

### CODE REVIEW

File: dll/win32/browseui/lang/ru-RU.rc [hunk 1]
```diff
@@ -326,11 +330,11 @@ END
...
-    IDS_SEARCH_FOLDER "Searching %s"
-    IDS_SEARCH_FILES_FOUND "%u files found"
-    IDS_SEARCH_OPEN_FOLDER "Open Containing &Folder"
-    IDS_COL_NAME "Name"
-    IDS_COL_LOCATION "In Folder"
-    IDS_COL_RELEVANCE "Relevance"
+    IDS_SEARCHINVALID "'%1' не является допустимой папкой."
+    IDS_SEARCH_FOLDER "Ищем в ""%s"""
+    IDS_SEARCH_FILES_FOUND "Найдено файлов: %u"
+    IDS_SEARCH_OPEN_FOLDER "Открыть папку с содер&жимым"
```
Comment (binarymaster) [inline dll/win32/browseui/lang/ru-RU.rc]:
```suggestion
    IDS_SEARCH_OPEN_FOLDER "Открыть &расположение файла"
```

File: dll/win32/browseui/lang/ru-RU.rc [hunk 2]
```diff
@@ -326,11 +330,11 @@ END
...
+    IDS_COL_NAME "Имя"
+    IDS_COL_LOCATION "Папка"
```
Comment (binarymaster) [inline dll/win32/browseui/lang/ru-RU.rc]:
```suggestion
    IDS_COL_LOCATION "В папке"
```

File: dll/win32/browseui/lang/ru-RU.rc [hunk 3]
```diff
@@ -1,4 +1,8 @@
-/* Russian language resource file by Petr Akhlamov (petr-akhlamov@yandex.ru) */
+/*
```
Comment (binarymaster) [inline dll/win32/browseui/lang/ru-RU.rc]:
```suggestion
/*
 * PROJECT:     ReactOS Explorer
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
 * PURPOSE:     Russian resource file
 * COPYRIGHT:   Copyright 2014-2015 Petr Akhlamov (petr-akhlamov@yandex.ru)
 *              Copyright 2017 Stanislav Motylkov (binarymaster@mail.ru)
 *              Copyright 2019 Abdulganiev Rafael (abdulganiev.rafael[at]gmail.com)
 */
```

File: dll/win32/browseui/lang/ru-RU.rc [hunk 4]
```diff
@@ -1,4 +1,15 @@
...
+ * PROJECT:     ReactOS Explorer
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Russian resource file
+ * COPYRIGHT:   Copyright 2014-2015 Petr Akhlamov (petr-akhlamov@yandex.ru)
+ *              Copyright 2017 Stanislav Motylkov (binarymaster@mail.ru)
+ *              Copyright 2019 Abdulganiev Rafael (abdulganiev.rafael[at]gmail.com)
+ */
+	Translators:
```
Comment (Mas4hmad) [inline dll/win32/browseui/lang/ru-RU.rc]: this line until line 12 make the build fail, you can delete it

**Pull request merged**